doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -139,16 +141,61 @@ all its clients.
 Steps to Enable mClock Profile
 ==============================
 
-The following sections outline the steps required to enable a mclock profile.
+As already mentioned, the default mclock profile is set to *high_client_ops*.
+The other values for the built-in profiles include *balanced* and
+*high_recovery_ops*.
+
+If there is a requirement to change the default profile, then the option
```

Comment:
I'd leave out the ceph.conf mentions, we want to encourage users to use 'ceph config' instead

Reply:
> I'd leave out the ceph.conf mentions, we want to encourage users to use 'ceph config' instead

Sure, I have remove reference to ceph.conf here and in one more place.